uui/components/datePickers/RangeDatePicker.tsx

```diff
@@ -1,118 +1,144 @@
-                    value={ this.state.inputValue.from }
-                    onValueChange={ this.getChangeHandler('from') }
-                    isInvalid={ this.props.isInvalid }
-                    isDisabled={ this.props.isDisabled }
-                    isReadonly={ this.props.isReadonly }
-                    onFocus={ (event) => this.handleFocus(event, 'from') }
-                    onBlur={ (event) => this.handleBlur(event, 'from') }
-                    isDropdown={ false }
-                    rawProps={ this.props.rawProps?.from }
-                    id={ this.props?.id }
-                />
-                <div className={ css.separator } />
-                <TextInput
-                    cx={ cx(css.dateInput, css['size-' + (this.props.size || 36)], this.state.inFocus === 'to' && uuiMod.focus) }
-                    placeholder={ this.props.getPlaceholder ? this.props.getPlaceholder('to') : i18n.rangeDatePicker.pickerPlaceholderTo }
-                    size={ this.props.size || '36' }
-                    value={ this.state.inputValue.to }
-                    onCancel={ this.props.disableClear ? null : this.state.inputValue.from && this.state.inputValue.to && this.handleCancel }
-                    onValueChange={ this.getChangeHandler('to') }
-                    isInvalid={ this.props.isInvalid }
-                    isDisabled={ this.props.isDisabled }
-                    isReadonly={ this.props.isReadonly }
-                    onFocus={ (e) => this.handleFocus(e, 'to') }
-                    onBlur={ (e) => this.handleBlur(e, 'to') }
-                    isDropdown={ false }
-                    rawProps={ this.props.rawProps?.to }
-                />
-            </div>
-        );
     };
+
+    return (
+        <Dropdown
+            renderTarget={ (renderProps) => {
+                return props.renderTarget?.(renderProps) || (
+                    <RangeDatePickerInput
+                        ref={ renderProps.ref }
+                        cx={ props.inputCx }
+                        onClick={ renderProps.onClick }
+                        onBlur={ onInputWrapperBlur }
+                        isDisabled={ props.isDisabled }
+                        isInvalid={ props.isInvalid }
+                        isReadonly={ props.isReadonly }
+                        size={ props.size }
+                        getPlaceholder={ props.getPlaceholder }
+                        disableClear={ props.disableClear }
+                        rawProps={ props.rawProps }
+                        inFocus={ inFocus }
+                        value={ inputValue }
+                        format={ format }
+                        onValueChange={ (i) =>{
+                            setInputValue(i);
+                        } }
+                        onInputFocus={ (event, inputType) => {
```

Comment:
Can we rename this props to just `onFocus` and `onBlur`, because it's obviously that this evens related to the input